plugin-taskbar/lxqttaskbar.cpp

```diff
@@ -432,12 +429,19 @@ void LXQtTaskBar::settingsChanged()
     // Delete all groups if grouping feature toggled and start over
     if (groupingEnabledOld != mGroupingEnabled)
     {
-        Q_FOREACH (LXQtTaskGroup *group, mGroupsHash.values())
+        QList<LXQtTaskGroup*> group_list;
```

Comment:
Because removing the widget from layou will change/invalidate the index into the layout....